internal/relay/relayer.go

```diff
@@ -73,6 +73,7 @@ func (r *Relayer) Run(ctx context.Context, tasks <-chan neutrontypes.RegisteredQ
 		)
 		select {
 		case query := <-tasks:
+			neutronmetrics.SetSubscriberTaskQueueNumElements(len(tasks))
```

Comment:
ah, I see the same track in the Subscriber. what's the point of two of them? isn't one enough measuring the len just before sending to the queue? also, as far as I can see, both tracks will have the same result since they will occur in the same time — on read from channel

Reply:
I have no idea how Golang scheduling works under the hood, hence I have placed calls to metrics in both places. What do you think is wise to do in this situation?